api/resolvers/sub.js

```diff
@@ -238,6 +254,21 @@ export default {
       if (typeof sub.ncomments !== 'undefined') {
         return sub.ncomments
       }
+    },
+    meSubscription: async (sub, args, { me, models }) => {
```

Comment:
This is called for every item displayed on a page, right? Assuming prisma isn't batching these, this is one roundtrip to the DB for *every* item on a page. 

I suggest looking at how `meMuteSub` is populated and mirroring that or removing territory subscriptions from the overflow on items.

On that note, I don't think territory subscriptions belong in the item overflow. The overflow is for: extra stuff as it relates to that item, or actions someone might urgently need (muting or